benchmarks/benchmark_autoround.py

```diff
@@ -0,0 +1,223 @@
+import argparse
```

Comment:
`bencmarks` is typically where we put microbenchmark for some quant technique (e.g. int4 weight only, fp6) for single linear kernels, I'm wondering if you could just add auto-round in https://github.com/pytorch/ao/blob/main/torchao/_models/llama/generate.py and https://github.com/pytorch/ao/blob/main/torchao/_models/llama/eval.py?

Reply:
Thanks for clarifying. I added the `auto-round` to [generate.py ](https://github.com/pytorch/ao/blob/main/torchao/_models/llama/generate.py) for benchmarking. Currently, I prefer using `eval_autoround.py` to evaluate the end-to-end accuracy of `lm-eval` tasks. It's adapted from [hf_eval](https://github.com/pytorch/ao/blob/main/scripts/hf_eval.py), which integrates the models from the Transformers more effectively. Please let me know if this makes sense to you.